.github/pull_request_template.md

```diff
@@ -9,11 +9,29 @@
 ## Issue link
 <!-- Add a link to the issue -->
 
-## Checklist before requesting a review
-- [ ] I have performed a self-review of my code
-- [ ] I have added thorough, applicable tests that validate my changes
+## Pre-review checklist
+
+If some steps are not applicable or in your judgment not necessary, remove and explain why rather than leave checkmarks blank.
+
+- [ ] I've tested the code as an end user would and included appropriate testing steps 
+- [ ] For all new functionality, I have added thorough, applicable tests that validate my changes
+- [ ] For improvements to existing functionality, I've followed the [campsite rule](https://www.stepsize.com/blog/how-to-be-an-effective-boy-girl-scout-engineer), leaving it a little better than I found it. 
```

Comment:
Only comment I have here is that we carved out the space in [dev standards](https://github.com/encodium/.github-private/pull/10/files#diff-2da9e88ad3cdfaa3c79ea878b10b11868483401013f7450596c8e4dba942f143R39) to talk about this more in detail but we're not linking to that page here.